Makefile

```diff
@@ -53,9 +53,15 @@ setup: clean version
 	source ~/.bash_profile && \
 	conda activate base && \
 	conda remove -n mcfacts-dev --all -y && \
-	conda create --name mcfacts-dev "python>=3.10.4<=3.13" pip "pytest" -c conda-forge -c defaults -y && \
+	conda create --name mcfacts-dev "python>=3.10.4,<=3.13" pip "pytest" -c conda-forge -c defaults -y && \
```

Comment:
Should be <3.13

3.13 does not work right now with the default conda installation.